### ISSUE TITLE
EE step on rave in the red woods (cannot complete ee)

### ISSUE THREAD

**Issue by Jkayz05** [auroramod/iw7-mod#124]
Issue with Easter Egg step on rave in the red woods upon completing the final trial getting headshots you're asked to hit a button in the basement of the spawn area, when hitting that button game closes and crashes to desktop with no error messages. currently no work around and the Easter Egg cannot be completed. i've tried different versions of the IW7 mod, older versions and newest version all have the same issue i'm not aware of anyone else is running into this issue 

https://cdn.discordapp.com/attachments/1118470197393764453/1212099214885257266/image.png?ex=65f09a5c&is=65de255c&hm=8513ffbbdde33c5ed6da2f864e7af3003221a0ed8d187f28f35e27cd6d17b666&

**Comment by mjkzy** [comment]
will check this one out

**Comment by Mart-inger** [comment]
I had the same issue and what fixed it for me was not using the type-2 gun. I tried the easter egg 3 times with the type-2 and it crashed all the times meanwhile when I didn’t buy it, the game never crashed(tried 5 times all 5 worked)

**Comment by Jkayz05** [comment]
yeah i never by the type 2 and still crashed for me

**Comment by Mart-inger** [comment]
Yes i was able to complete it with and without directors cut multiple times in 3 different versions, the only problem was when I was using the type-2😔.

I did them on a server and not private match,maybe its that but it still crashes when buying the type-2.

**Comment by mjkzy** [comment]
i need more information about this. im not much of a IW7 player, sorry :P

is the type-2 a weapon? if so, what happens that can't make the EE complete? is there a crash? if so, is there a crash dump? that could help us find the issue

**Comment by Jkayz05** [comment]
yes the type 2 is a gun, all i know is that on the updated version of iw7 it crashes when doing the rave ee i found a version that works tho it version 89eac38 that version doesnt crash on the rave ee but the versions after that one do i can have a look for a crash file but it never gave me any errors just straight to desktop

**Comment by RoeanAigis** [comment]
This issue still happens currently on the latest commit making it impossible to complete directors cut without using console commands.

**Comment by OctoHenry** [comment]
This just happened to me, and I didn't have the type 2. Massive bummer

**Comment by skkuull** [comment]
Idk really where the issue is, i just completed the easter egg normally and nothing happened throughout the game play
